src/parser/jobs/gnb/modules/Ammo.tsx

```diff
@@ -170,6 +175,18 @@ export default class Ammo extends Module {
 			value: this.erroneousCircles,
 		}))
 
+		this.suggestions.add(new TieredSuggestion({
```

Comment:
Future TODO, might be worth attempting to figure out why they had extra ammo. One is unavoidable sometimes and a minor on it is a bit eh. I don't think the other tanks (or most DPS) ping people for leftover gauge etc so it stands out a bit.

Reply:
I'll have to think on this. Currently the idea behind this is that Ammo GCDs > Skill GCDs, so they should only have leftover ammo if: 
a) They didn't use ammo and just used a weaponskill
b) had 2 ammo before entering a ammo combo and couldn't get to use the second ammo
c) they popped bloodfest like literally before the thing died in vain

The reason one ammo leftover is set to minor is it's on a range from 300-100 potency lost, where as 2 ammo is 800-600 based on what weaponskill they used instead of it. (possibly more if they tossed away an ammo combo, but we yell elsewhere for that mistake)